### DISCUSSION TITLE
Automatic return from folder button  to profile page

### DISCUSSION THREAD

**Discussion by UweSchwo** [elgatosf/elgato#64] [Ideas]
Dear developers, please give every button in sub folders a "Return" check mark  in order to optionally save one button press to return to the profile page.

This would be so much easier than create a multi action for every button.

**Comment by GeekyEggo** [comment]
Hey @UweSchwo, thanks for your feedback. With our GitHub discussions designed for our SDK, we do ask that any feedback relating to our software, e.g. Stream Deck, be submitted via our [Submit a Feature Request](https://help.elgato.com/hc/en-us/articles/360060806532--Submit-a-Feature-Request-) page. This will ensure it reaches the right team. You can also refer to our [website](https://www.elgato.com) or [subreddit](https://www.reddit.com/r/elgato/) to see the latest product releases and software updates.
